apps/starter/src/ensemble/dashboard.yaml

```diff
@@ -265,14 +265,149 @@ View:
         - Text:
             id: myText
             text: Peter Parker
-        - Button:
-            label: Click Me
-            onTap:
-              executeCode: myText.setText("Spiderman")
         - Text:
             text: ${myText.text}
         - Text:
             text: ${getDummyJson.body.total}
+        - Stepper:
+            id: stepper1
+            styles:
+              connectorColor: "#31C48D"
+              connectorHeight: 4
+            steps:
+              - stepLabel: Match
+                contentWidget:
+                  Column:
+                    crossAxis: center
+                    mainAxis: spaceBetween
+                    children:
+                      - Text:
+                          text: Hello
+                      - Button:
+                          label: Doing Good
+              - stepLabel: SubmitDocs
+                contentWidget:
+                  Column:
+                    crossAxis: center
+                    mainAxis: spaceBetween
+                    children:
+                      - Text:
+                          text: Doing asd
+                      - Button:
+                          label: Doing Good
+              - stepLabel: Check
+                contentWidget:
+                  Column:
+                    crossAxis: center
+                    mainAxis: spaceBetween
+                    children:
+                      - Text:
+                          text: Doing asd
+                      - Button:
+                          label: Doing Good
+            item-template:
+              data:
+                - name: Match
+                  iconName: ScaleOutlined
+                - name: SubmitDocs
+                  iconName: FindInPageOutlined
+                - name: Review
+                  iconName: Check
+                - name: Enroll
+                  iconName: Search
+              name: step
+              template:
+                Conditional:
+                  conditions:
+                    - if: ${index < stepsLength - 1}
```

Comment:
Why is this check needed?

Reply:
this check is for the final step, this yaml can be improved but for now it checks to make sure final step goes to else condition so that its shown in only two states inactive, completed.

Reply:
Okay this is fine for now then